### PR TITLE
feat: use js-ipfs 0.30 and aegir lint

### PR DESCRIPTION
**Pull request by daviddias**
Depends on:
- [x] https://github.com/ipfs-shipyard/y-ipfs-connector/pull/14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
     "remark": "^8.0.0",
     "remark-html": "^6.0.1",
     "remark-html-katex": "^1.0.2",
     "remark-math": "^1.0.1",
-    "safe-buffer": "^5.1.1",
     "y-array": "^10.1.4",
-    "y-indexeddb-encrypted": "^0.0.1",
-    "y-ipfs-connector": "^2.2.0",
+    "y-indexeddb-encrypted": "~0.0.1",
+    "y-ipfs-connector": "ipfs-shipyard/y-ipfs-connector#js-ipfs0.30",
```
Comment (daviddias) [inline package.json]: - [ ] Change this before merge

**Comment by daviddias** [comment]
@pgte ready for review, merge and publish too

**Review by pgte — APPROVED** [review]
LGTM

**Comment by daviddias** [comment]
@pgte you forgot to publish

**Comment by pgte** [comment]
Landed on v0.13.0

**Pull request merged**